msg/en_US/csql.msg

```diff
@@ -188,6 +190,12 @@ $ csql help messages
    ;HISTORYRead <history_num>   - read entry on the history number into command buffer.\n\
    ;TRAce [ON/OFF] [text/json]  - enable/disable sql auto trace.\n\
    ;SIngleline [ON|OFF]         - enable/disable single-line mode.\n\
+   ;Connect [username] [dbname]@[hostname]\n\
+                                - connect user, database, host.\n\ 
+   ;.Hist [ON/OFF]              - start/stop collecting statistics information in CSQL.(DBA only)\n\
+   ;.Clear_hist                 - clear the CSQL statistics information in the buffer.\n\
+   ;.Dump_hist                  - display the CSQL statistics information in CSQL.\n\
+   ;.X_hist                     - display the CSQL statistics information in CSQL with statistics data initialized.\n\
```

Comment:
add (DBA only)

Reply:
I tried but except DBA it works. which one is should I add DBA only?

Reply:
I asked this Chanho and then he recommend 'available' most.

'effective' and 'active' are recommended lower priority.